compiler/parser.cpp

```diff
@@ -5428,10 +5436,6 @@ static void statement(int *lastindent,int allow_decl)
     doassert();
     lastst=tASSERT;
     break;
-  case tSLEEP:

```

Comment:
Did you mean to slip this removal into this commit?

Reply:
Yes, indeed.